server/lists/adapters/workstacks.js

```diff
@@ -188,6 +189,9 @@ const bindDisplayElements = fromStaticList => async (stage) => {
 
         if (primaryCorrespondent) {
             stage.primaryCorrespondentAndRefDisplay.primaryCorrespondentFullName = primaryCorrespondent.fullname;
+            if (stage.caseType === 'FOI') {
```

Comment:
We should have a shapshot test for the instance when the primaryCorrespondentFullName is provided.